qmd/sets.qmd

```diff
@@ -4,67 +4,84 @@ The list of sets in aphabetical order of setsID
 
 
 ```{r, echo=FALSE}
+# Utility Functions
+handle_empty_character_vector <-
+  function(check_vector, replace_string) {
+    if (length(check_vector) < 1) {
+      return(replace_string)
+    }else{
+      return(check_vector)
+    }
+  }
+
+list_creation <- function(string_to_append, list_values){
```

Comment:
Rename to `create_html_list`. Functions should start with verbs.

Reply:
I don't think the appending needs to be done in this function i.e. we don't need the `string_to_append` parameter,

A function should try to have only one responsibility and there's no need for this function to have the responsibility of the appending.

Reply:
Add Roxygen comment.